
class PasswordValidationError(Exception):
    pass

class AuthenticationFailedError(Exception):
    pass